torchaudio/models/wav2vec2/components.py

```diff
@@ -314,7 +318,185 @@ def forward(
         output = output.transpose(2, 1).reshape(batch_size, length, embed_dim)
 
         output = self.out_proj(output)
-        return output
+        return output, None  # Necessary for compatibility with WavLMSelAttention
+
+
+class WavLMSelfAttention(nn.Module):
+    """Multi-headed self-attention for WavLM model :cite:`wavlm2021`.
+    Source: https://github.com/microsoft/UniSpeech/blob/2e9dde8bf815a5f5fd958e3435e5641f59f96928/WavLM/modules.py
+
+    Args:
+        embed_dim: Total dimension of the model.
+        num_heads: The number of heads.
+        dropout: Dropout probability on attn_output_weights.
+        bias: If ``True``, add bias to projections for queries and values.
```

Comment:
for argument with default value, the type will be `(type, optional)`. could you also add the default value at the end of docstring?
`bias (bool, optional): If ``True``, add bias to projections for queries and values. (Default: ``True``)`